functions/functions/ratingCalculator.js

```diff
@@ -0,0 +1,42 @@
+"use strict";
+
+const admin = require('firebase-admin');
+const functions = require("firebase-functions");
+const db = admin.firestore();
+
+const average = (array) => array.reduce((a, b) => a + b) / array.length;
+
+// Keeps track of the length of the "waypoints" child list in a separate property.
```

Comment:
Ich glaub hier sollte sowas stehen wie "Calculate average of ratings for a route" ;)